src/game/server/gamecontext.cpp

```diff
@@ -713,16 +713,32 @@ void CGameContext::OnClientConnected(int ClientID)
 bool CGameContext::OnClientDrop(int ClientID, const char *pReason, bool Force)
 {
 	m_pController->t_stats->on_drop(ClientID, pReason);
-	
-	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force) return false;
 
+	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force)
+	{
+		if(m_Ragequit <= 1)
+			m_Ragequit++;
+		if(m_Ragequit == 1)
```

Comment:
Should this be 1? Maybe a higher number depending on the time?

Reply:
no u